sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_conversation_app_async.py

```diff
@@ -30,47 +32,65 @@ async def sample_analyze_conversation_app_async():
     from azure.core.credentials import AzureKeyCredential
 
     from azure.ai.language.conversations.aio import ConversationAnalysisClient
-    from azure.ai.language.conversations.models import ConversationAnalysisOptions
+    from azure.ai.language.conversations.models import DateTimeResolution
 
     # get secrets
-    conv_endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    conv_project = os.environ["AZURE_CONVERSATIONS_PROJECT"]
-
-    # prepare data
-    query = "One california maki please."
-    input = ConversationAnalysisOptions(
-        query=query
-    )
-
-    # analyze query
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_CONVERSATIONS_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_CONVERSATIONS_DEPLOYMENT_NAME"]
+
+    # analyze quey
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=conv_project,
-            deployment_name='production'
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = await client.analyze_conversation(
+            body={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
```

Comment:
What do I set for an ID value?

Reply:
user id

Reply:
What I meant is - where do I get this value from?
Do I just use an arbitrary value? Is it something I should know beforehand? If so, where do I get it from?